generators/generator-base-blueprint.js

```diff
@@ -95,4 +106,14 @@ module.exports = class extends BaseGenerator {
         }
         return useBlueprints;
     }
+
+    removeConfigDuplicates() {
+        Object.keys(this.config.getAll()).forEach(key => {
```

Comment:
For the blueprint generators the properties / configuration will remain inside the blueprint's key rather than inside the 'generator-jhipster` key. 

So iterating like this will not have an impact. Since there will not be any common property between them.

Reply:
For client/server it works because app module doesn't have blueprint support.

I think that Kotlin is different.

I will try copying every config the first execution.

Reply:
@sendilkumarn What about now?